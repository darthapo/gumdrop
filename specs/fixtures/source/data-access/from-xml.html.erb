<ul>
<% data.xml_data.diet.food.each do |food|%>
<li><%= food.name %></li>
<% end %>
</ul>