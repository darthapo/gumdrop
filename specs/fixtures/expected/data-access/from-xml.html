<body class="site"><ul>
<li>Bland</li>
<li>More Bland</li>
</ul></body>